### PR TITLE
Aksel v7

### PR DESCRIPTION
**Pull request by HalvorHaugan**

### CODE REVIEW

File: .changeset/little-files-cough.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-react": major
+---
+
+Tooltip: :boom: Updated labeling of items inside Tooltip. See migrations for how to update. TODO: Insert link here.
```
Comment (KenAJoh) [inline .changeset/little-files-cough.md]:
```suggestion
Tooltip: :boom: Updated labeling of items inside Tooltip. [See migrations for how to update](https://aksel.nav.no/grunnleggende/kode/migrering#3b5cf05fd100). 
```

File: .changeset/small-apples-pretend.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/aksel-icons": major
+---
+
+Icons: Removed renamed icons. See migration for changes. TODO: add migration.
```
Comment (KenAJoh) [inline .changeset/small-apples-pretend.md]:
```suggestion
Icons: Removed renamed icons. [See migration for changes](https://aksel.nav.no/grunnleggende/kode/migrering#194b60833d9e).
```

File: .changeset/stupid-pianos-end.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-tailwind": major
+---
+
+Tailwind: Extended 'screens'-config in theme to match Aksel breakpoints. Tailwind and Primitives can now be used side by side with matching dynamic breakpoints. See migration for potential issues when adopting. TODO: Add link
```
Comment (KenAJoh) [inline .changeset/stupid-pianos-end.md]:
```suggestion
Tailwind: Extended 'screens'-config in theme to match Aksel breakpoints. Tailwind and Primitives can now be used side by side with matching dynamic breakpoints. [See migration for potential issues when adopting](https://aksel.nav.no/grunnleggende/kode/migrering#3a2340f6f69b).
```

File: .changeset/little-files-cough.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-react": major
+---
+
+Tooltip: :boom: Updated labeling of items inside Tooltip. [See migrations for how to update](https://aksel.nav.no/grunnleggende/kode/migrering#3b5cf05fd100).
```
Comment (HalvorHaugan) [inline .changeset/little-files-cough.md]: Should we say "migration **guide**"?
```suggestion
Tooltip: :boom: Updated labeling of items inside Tooltip. [See migration guide for how to update](https://aksel.nav.no/grunnleggende/kode/migrering#3b5cf05fd100).
```
Or maybe just:
```suggestion
Tooltip: :boom: Updated labeling of items inside Tooltip. [How to update](https://aksel.nav.no/grunnleggende/kode/migrering#3b5cf05fd100). 
```

**Review by KenAJoh — DISMISSED** [review]

File: @navikt/core/react/src/form/error-summary/ErrorSummary.tsx
Comment (HalvorHaugan) [inline @navikt/core/react/src/form/error-summary/ErrorSummary.tsx:1]: Should we omit tabIndex?

Comment (KenAJoh) [reply @navikt/core/react/src/form/error-summary/ErrorSummary.tsx:1]: Sure, omitted it now 👍

**Pull request merged**